src/components/App.jsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import Layout from './Layout/Layout'
+import * as S from '../Style'
+const App = () => {
+  return (
+    <>
```

Comment:
to normalne w React, takie puste tagi?

Reply:
Tak, jeżeli nie masz jakiegoś elementu wrapujacego